std/assembly/string.ts

```diff
@@ -641,19 +641,21 @@ export namespace String {
       while (strOff < strEnd) {
         let c1 = <u32>load<u16>(strOff);
         if (c1 < 128) {
-          if (nullTerminated && !c1) break;
-          bufLen += 1; strOff += 2;
+          // @ts-ignore: cast
+          if (nullTerminated & !c1) break;
```

Comment:
Can also do `u32(nullTerminated) & !c1` here to avoid the ts-ignore.

Reply:
Yes, and in first version used exactly that but later we decide just comment with ts ignorance)

Reply:
It was in other PR

Reply:
I see, hmm. Do you remember if that was due to wrapping?